src/Select/style.css

```diff
@@ -0,0 +1,108 @@
+.ant-select {
+  @apply font-inter;
```

Comment:
You can merge with the below rules. We usually use several `@apply` when the number of classes are too many.